controllers/compaction/config.go

```diff
@@ -30,15 +30,17 @@ var featureList = []featuregate.Feature{
 }
 
 const (
-	enableBackupCompactionFlagName = "enable-backup-compaction"
-	workersFlagName                = "compaction-workers"
-	eventsThresholdFlagName        = "etcd-events-threshold"
-	activeDeadlineDurationFlagName = "active-deadline-duration"
+	enableBackupCompactionFlagName    = "enable-backup-compaction"
+	workersFlagName                   = "compaction-workers"
+	eventsThresholdFlagName           = "etcd-events-threshold"
+	activeDeadlineDurationFlagName    = "active-deadline-duration"
+	metricsScrapeWaitDurationFlagname = "metrics-scrape-wait-duration"
 
-	defaultEnableBackupCompaction = false
-	defaultCompactionWorkers      = 3
-	defaultEventsThreshold        = 1000000
-	defaultActiveDeadlineDuration = 3 * time.Hour
+	defaultEnableBackupCompaction    = false
+	defaultCompactionWorkers         = 3
+	defaultEventsThreshold           = 1000000
+	defaultActiveDeadlineDuration    = 3 * time.Hour
+	defaultMetricsScrapeWaitDuration = 0
```

Comment:
Please update the release note as well as PR description

Reply:
Done